src/signed-xml.ts

```diff
@@ -154,6 +160,9 @@ export class SignedXml {
     this.CanonicalizationAlgorithms;
     this.HashAlgorithms;
     this.SignatureAlgorithms;
+    // this populates only after verifying the signature
+    // array of bytes that are cryptographically authenticated
+    this.signedReferences = []; // TODO: should we rename this to something better.
```

Comment:
That seems like a descriptive name to me. What other thoughts do you have?

Reply:
Chatty suggested options like `authenticatedData` and `signatureVerifiedBytes`. I have no problems with log descriptive names. I think, at a minimum, we should make sure the type is correct; right now it is untyped.